core/server/models/post.js

```diff
@@ -129,12 +186,12 @@ Post = ghostBookshelf.Model.extend({
 
         // ### Business logic for published_at and published_by
         // If the current status is 'published' and published_at is not set, set it to now
-        if (this.get('status') === 'published' && !this.get('published_at')) {
-            this.set('published_at', new Date());
+        if (newStatus === 'published' && !publishedAt) {
+            this.set('published_at', moment.utc().toDate());
```

Comment:
moment.utc().toDate() will (or should) not change the current behaviour, thats because every time you call .toDate on a moment object you will get the local machine time. so because our server is configured to UTC time, all your dates should be already in UTC in the database (that's what i expect) - tell me if i am wrong.

i need to admit this change wasn't needed and is little confusing (💃 )  and i am totally fine with revert, because if everybody knows that dates in application layer expect that machine is configured "correctly". 

may the next universe be without dates!